src/components/form-sections/HasNextEnrollment/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { isEmpty } from 'lodash';
+import { getNextEnrollment } from '../../../lib/services';
+
+const HasNextEnrollment = ({ enrollmentId }) => {
```

Comment:
an implicit naming convention is to append the style of the component to its name : HasNextEnrollmentNotification